### PR TITLE
Download the latest release of the downloader

### PR DESCRIPTION
**Pull request by Ao-senXiong**

### CODE REVIEW

**Comment by Ao-senXiong** [comment]
Prof. Dietl, will you make a new release on both checker-framework-languageserver-downloader and checker-framework-languageserver? I seem do not have permission to do it in this org. Otherwise, please grant me the write permission to the repo.

**Review by wmdietl — CHANGES_REQUESTED** [review]
Nice!
I see a few mentions of the old downloader jar file, e.g. here https://github.com/eisopux/checker-framework-vscode/blob/0ee64c5af51a8c43384312b10559512a8f844734/src/extension.ts#L103
Please do a search for it throughout, including the build instructions.

I'm wondering whether lower-case `download-downloader.sh` would be better. Or `get-downloader-release.sh`?

**Comment by Ao-senXiong** [comment]
> Nice! I see a few mentions of the old downloader jar file, e.g. here
> 
> https://github.com/eisopux/checker-framework-vscode/blob/0ee64c5af51a8c43384312b10559512a8f844734/src/extension.ts#L103
> 
> 
> Please do a search for it throughout, including the build instructions.
> I'm wondering whether lower-case `download-downloader.sh` would be better. Or `get-downloader-release.sh`?

Thanks, I will take get-downloader-release.sh. Do you want me to also update all version numbers from 0.2 to 0.3? Or you will do it when you make a new release?

**Comment by wmdietl** [comment]
> Thanks, I will take get-downloader-release.sh. Do you want me to also update all version numbers from 0.2 to 0.3? Or you will do it when you make a new release?

Ideally, we'll have only one place where we update the release number. If you see a way to do that, please make a PR for that. Otherwise, I'll do it with a release.

**Comment by Ao-senXiong** [comment]
> 

Ok, before you publish the extension in the vscode marketplace, I can try to test it after the new release of the downloader and language server. Just double check it is working.

File: package.json [hunk 1]
```diff
@@ -93,7 +93,8 @@
     }
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
+    "vscode:prepublish": "npm run compile; npm run download-downloader",
+    "download-downloader": "./scripts/get-downloader-release.sh",
```
Comment (wmdietl) [inline package.json:97]:
```suggestion
    "vscode:prepublish": "npm run compile; npm run get-downloader-release",
    "get-downloader-release": "./scripts/get-downloader-release.sh",
```

**Review by wmdietl — APPROVED** [review]
Thanks!

**Pull request merged**